### PR TITLE
Create README.md for Filecoin docs repo

### PR DESCRIPTION
**Pull request by dawnkelly09**
Based upon IPFS docs README located here: https://github.com/ipfs/ipfs-docs/blob/main/README.md

[readme.md](https://github.com/filecoin-project/filecoin-docs/files/8644070/readme.md)

There are a few unanswered questions in here. Please search for "TO-DO" to locate the comments on them.

### CODE REVIEW

**Comment by dawnkelly09** [comment]
@mishmosh Forgot to tag you when I opened this. I feel like I missed the steps of getting this connected to the related issue but I will study up on that for my next one.

**Comment by mishmosh** [comment]
Github uses [magic keywords to link PRs & issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue), so adding the following line will automatically link them:

Fixes #1248.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# filecoin-docs
```
Comment (mishmosh) [inline README.md:1]:
```suggestion
# Filecoin Documentation
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Filecoin Docs
```
Comment (mishmosh) [inline README.md:2]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+<!--TO-DO: where can I get assets for Filecoin to replace IPFS logo and Build, Made, Project Icons?-->
+
+![IPFS documentation logo.](ipfs-docs-header.png)
```
Comment (mishmosh) [inline README.md:7]: I think we can skip this header entirely, and just use text as in line 1.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+[![Build status icon.](https://img.shields.io/circleci/project/github/ipfs/ipfs-docs/master.svg?style=flat-square)](https://circleci.com/gh/ipfs/ipfs-docs)
```
Comment (mishmosh) [inline README.md:8]: This should be generated automatically. @johnnymatthews do you know how this is connected to circleci?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+[![Made by icon.](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai/)
+[![Project icon.](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+
+This repository organizes overall documentation issues across the Filecoin project.
+
+**If you'd just like to read the Filecoin docs, we recommend the website version of the info contained in this repo! https://docs.filecoin.io/**
+
+## Get involved
+
+<!--TO-DO: do we want to link to this same bounty page or is there a Filecoin specific one?-->
```
Comment (mishmosh) [inline README.md:18]: There is no active bounty program, so let's comment out everything related to bounties (part of line 20, plus everything in the Bounties section beginning line 26).

**Review by mishmosh — CHANGES_REQUESTED** [review]
Overall, nice start! Some comments inline. There are also some procedural/build-related questions that @johnnymatthews should answer instead of me.

File: README.md (hunk 4 above)
Comment (johnnymatthews) [reply README.md:7]: We don't need to use the _Build status icon_ (that's a legacy thing in IPFS docs that we should actually remove).

```suggestion
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+If you find something wrong within this repository, please raise an [issue here →](https://github.com/filecoin-project/filecoin-docs/issues). Unless the issue is urgent, updates will be batch-merged into `main` on Tuesdays or Thursdays.
+
+### Bounties
+
+<!--TO-DO: do we want to link to this same bounty page or is there a Filecoin specific one?-->
+
+You can earn the undying love of the Filecoin community _and_ make some money by closing an issue with the [`bounty` tag](https://github.com/ipfs/ipfs-docs/issues?q=is%3Aopen+is%3Aissue+label%3Abounty)! Submissions must be production-ready and meet all the specifications listed on the issue page. Check out the [current list of open bounties →](https://github.com/ipfs/devgrants/projects/1)
+
+If you are attempting to close an issue, great! Thanks for the help! Please leave a comment within the issue requesting to be assigned to that issue **before** submitting a pull request. This minimizes the chance of multiple different contributors duplicating work by submitting pull requests for the same issue. If you submit a pull request to an issue _without_ first being assigned to it, that pull request may not be accepted.
+
```
Comment (johnnymatthews) [inline README.md:32]:
```suggestion
```

We don't have a bounty program for Filecoin Docs right now. Let's remove this.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+1. Close the local server with `CTRL` + `c`.
+1. To restart the local server, run `npm start` from within the `filecoin-docs` folder.
+
+## Core members
+
+<!--TO-DO: keep these same key members or update for more Filecoin specific people?-->
+
+- [@johnnymatthews](https://github.com/johnnymatthews): Project leadership, organization, and primary contact
+- [@cwaring](https://github.com/cwaring): Development support
+
```
Comment (johnnymatthews) [inline README.md:74]:
```suggestion
```

This section changes far too often, so let's just remove it for now.

**Review by johnnymatthews — APPROVED** [review]
Made a couple of tiny changes, but yeah this is good to merge!

**Pull request merged**